src/adagrams.js

```diff
@@ -1,8 +1,84 @@
+// Distribution of Letters
+const letterPool = ['A','A','A','A','A','A','A','A','A','B','B','C','C','D','D','D','D',
+                    'E','E','E','E','E','E','E','E','E','E','E','E',
+                    'F','F','G','G','G','H','H','I','I','I','I','I','I','I','I','I',
+                    'J','K','L','L','L','L','M','M','N','N','N','N','N','N',
+                    'O','O','O','O','O','O','O','O','P','P','Q','R','S','S','S','S',
+                    'T','T','T','T','T','T','U','U','U','U','V','V','W','W',
+                    'X','Y','Y','Z']
+
+const scoredWords = {};
+scoredWords['A'] = 1;
+scoredWords['B'] = 3;
+scoredWords['C'] = 3;
+scoredWords['D'] = 2;
+scoredWords['E'] = 1;
+scoredWords['F'] = 4;
+scoredWords['G'] = 2;
+scoredWords['H'] = 4;
+scoredWords['I'] = 1;
+scoredWords['J'] = 8;
+scoredWords['K'] = 5;
+scoredWords['L'] = 1;
+scoredWords['M'] = 3;
+scoredWords['N'] = 1;
+scoredWords['O'] = 1;
+scoredWords['P'] = 3;
+scoredWords['Q'] = 10;
+scoredWords['R'] = 1;
+scoredWords['S'] = 1;
+scoredWords['T'] = 1;
+scoredWords['U'] = 1;
+scoredWords['V'] = 4;
+scoredWords['W'] = 4;
+scoredWords['X'] = 8;
+scoredWords['Y'] = 4;
+scoredWords['Z'] = 10;
```

Comment:
These lines of code (`const letterPool = ...`, `const scoredWords = {};`, `scoredWords['A']`, etc...) are lines of assignment. I never want these lines of assignment outside of a function! It would be best to move these into the `drawLetters` function and `scoreWord` function, even if it makes those functions really big. In some cases, like in script files that are attached to HTML, I may expect code like this, but otherwise in this assignment I did not expect that-- we wanted the `Adagrams` object to be self contained.